src/conv.jl

```diff
@@ -36,8 +53,14 @@ function crosscor(x::A, w::A; pad = 0, stride = 1, dilation = 1) where A<:Abstra
         x, w, pad = pad_, stride = stride_, dilation = dilation)
 end
 
-∇conv_data(dy::A, x::A, w::A; pad = 0, stride = 1, dilation = 1, flipkernel = 0) where A<:AbstractArray =
-  ∇conv_data!(zero(x), dy, x, w; pad = pad, stride = stride, dilation = dilation, flipkernel=flipkernel)
+function ∇conv_data(dy::A, w::A, x_dims=nothing; pad = 0, stride = 1, dilation = 1, flipkernel = 0) where A<:AbstractArray
```

Comment:
Couple small things
* Can you make this a keyword argument `size`
* Can you add a simple three-arg wrapper with a deprecation warning
* You should compare `x_dims === nothing` so that type inference can remove the check

Reply:
When `size` method is being used on `conv`, it gives `MethodError` if this argument is named `size`. How about using `dims` instead?